### DISCUSSION TITLE
Possible documentation bug (README.md, IrSender.begin)?

### DISCUSSION THREAD

**Discussion by Openuser87** [Arduino-IRremote/Arduino-IRremote#1186] [Q&A] [answered]
README.md states:

> For sending, call `IrSender.begin(ENABLE_LED_FEEDBACK);` or `IrSender.begin(DISABLE_LED_FEEDBACK);` in setup().

Further it's stated that you must use the 3-argument syntax **if** `IR_SEND_PIN` is not defined.

But I could only get the 3-argument version to run, though I did define `IR_SEND_PIN`. When compiling with a single argument (e.g. `IrSender.begin(ENABLE_LED_FEEDBACK);`), the sender is doing nothing. I solved it just by chance (coming across [this](https://forum.arduino.cc/t/ir-transmitter-not-working-with-arduino/856655), although the "2-argument" syntax there didn't work with me either, but made me think in the right direction).

Maybe I am misunderstanding something, or this needs correction? Might save folks some time in the latter case...

**Comment by ArminJo** [comment, marked as answer]
Have defined IR_SEND_PIN before the line `#include <IRremote.hpp>`?
Have you read [this](https://github.com/Arduino-IRremote/Arduino-IRremote#compile-options--macros-for-this-library)?

**Reply by Openuser87** [reply]
No and no. :upside_down_face: 

Thank you, just tested it - when defined before `#include`, it works! :+1: Sorry, not a programmer :wink: 

So, everything perfect then. Thanks for the quick reply and for the excellent library!
